### PR TITLE
Angular support

### PR DESCRIPTION
**Pull request by edusperoni**
Fixes https://github.com/bradleygore/nativescript-textinputlayout/issues/28

tested on both Android and iOS with webpack aot and uglify.

remember to build it with `npm run build` so the ngc kicks in

### CODE REVIEW

File: angular/valueacessor.directive.d.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { TextValueAccessor } from "nativescript-angular/forms/value-accessors/text-value-accessor";
+/**
+ * The accessor for setting a selectedIndex and listening to changes that is used by the
+ * {@link NgModel} directives.
+ *
+ *  ### Example
+ *  ```
+ *  <MaskedTextField [(ngModel)]="model.test">
```
Comment (bradleygore) [inline angular/valueacessor.directive.d.ts]: Don't think this example should reference MaskedTextField

Comment (edusperoni) [reply angular/valueacessor.directive.d.ts]: ooops, my bad. I'll fix it tomorrow. I borrowed part of the doc from the maskedtextfield

**Comment by bradleygore** [comment]
@edusperoni - I really haven't been using {N} for a bit now (clients had other needs than mobile at this time) so I think one of the below is what I'd prefer:

One of your ideas from above:
> or just the angular bindings with your plugin as a dependency

Or, if you want to be added as a collaborator to help maintain the plugin (specifically the angular-specific bits) we could go that route and merge the PR into here.

Thoughts?

**Comment by edusperoni** [comment]
I'm fine with being added as a contributor to handle the angular bits, but I'll test the separate plugin structure next week and get back to you to make sure it's the best solution

**Comment by bradleygore** [comment]
@edusperoni  Given there's a work-around that is super easy (using text binding and on change binding) that allows us to not have to put angular-specific stuff into this plugin, I'm declining this PR. The effort is greatly appreciated, however!

**Pull request closed without merging**